src/sentry/static/sentry/app/views/projects/projectContext.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import createReactClass from 'create-react-class';
 import Reflux from 'reflux';
 import DocumentTitle from 'react-document-title';
-import {withRouter} from 'react-router';
+import {withRouter, browserHistory} from 'react-router';
```

Comment:
fyi you can use browserHistory via `props.router.history`

Reply:
You mean `this.context.router` right?

Reply:
`withRouter` HoC injects `router` as a prop

Reply:
Ahh got you, I was just looking elsewhere and didn't see anything else like this.